packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -772,6 +775,12 @@ export class DeltaManager
             type,
         };
 
+        if (type === MessageType.NoOp) {
```

Comment:
I think (though I'd double check) we can be more generic here, and check for all system ops (isSystemMessage).
90% of them should not be sent by client, but noop & summarize are in the list.
I think it's good change, given that Container.submitContainerMessage() validates that container can only sent 3 types of messages (i.e. should not be able to sent system ops).

Reply:
I think it is more about what ops server may not acknowledge and that are noops. Server should ack the summary op and if it is not acked then client should resend it. So I think noop check is fine?

Reply:
Can you please check if we indeed resend summary Op on reconnection if it was lost?
If we do, then yes, I agree with you (even though the point is a but murky, as another container runtime may have different behavior)

Reply:
As I checked summarizer never rejoins on disconnection. So this won't occur for summarize op.

Reply:
This might change in future. Either way - it's fine one way or another :)